python/ray/autoscaler/_private/kuberay/node_provider.py

```diff
@@ -142,7 +142,7 @@ def _url(self, path: str) -> str:
             + path
         )
 
-    def _get(self, path: str) -> Dict[str, Any]:
+    def get(self, path: str) -> Dict[str, Any]:
```

Comment:
ooc why this change?

Reply:
+1, I think keeping it private makes more sense :)

Reply:
It's currently used in `autoscaling_config.py` to fetch the RayCluster CR, therefore is not private at the moment.
I'll just use the Python K8s client there instead -- we don't need to see the node provider in that file.